src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultTag/styles.ts

```diff
@@ -0,0 +1,62 @@
+import styled from '@emotion/styled';
+import Icon from '../../../ignitus-Utilities/Components/icon';
+
+export const StyledIcon = styled(Icon)`
+  height: 1.2rem;
+  width: 1.2rem;
+  fill: black;
+  transition: 0.3s;
+  flex: 1;
+`;
+
+export const StyledTag1 = {
+  backgroundColor: "#FFA500",
+  color: "white",
```

Comment:
use white defined as const in [colors.ts](https://github.com/Ignitus/Ignitus-client/blob/master/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/colors.ts), same for background color as well, `font-weight, font-size` as well, and please don't use `-webkit`, we are not only targeting webkit-browsers.

Reply:
Done